.env

```diff
@@ -0,0 +1,11 @@
+MYSQLDB_DATABASE=book_store
+MYSQLDB_USER=mysql_user
+MYSQLDB_PASSWORD=password1234
+
+MYSQL_LOCAL_PORT=3306
+MYSQL_DOCKER_PORT=3306
+
+SPRING_LOCAL_PORT=8080
+SPRING_DOCKER_PORT=8080
+
+DEBUG_PORT=5005
```

Comment:
It is important to remember that pushing an `.env` file with credentials is not secure. The `.env` file should be added to `.gitignore`. Instead, create an `.env.template` file and declare the variable names without initializing them with values. Later, add recommendations for filling this file by the user of your application in `README.md`.